ci/testkomodo.sh

```diff
@@ -35,8 +35,7 @@ run_ert_with_opm() {
 }
 
 run_everest_tests() {
-    python -m pytest -n auto tests/everest -s \
-    --ignore-glob "*test_visualization_entry*"
+    python -m pytest -n auto tests/everest -s
```

Comment:
I guess this is not relevant anymore?

Reply:
`--ignore-glob "*test_visualization_entry*"` is not needed anymore I changed the test so that it uses a mocked plugin manager so we ar no longer dependent on what is installed in the environment.